Libbna/CUSTOM-CMS
# #245: Solved the issue with search bar responsiveness

* Added responsiveness to the search bar.

* Testing Steps : 
1. View the website on mobile or tablet view.
2. Open the hamburger menu.
3. The search bar should be properly styled and responsive.
4. Refer screenshot : 

![Screenshot from 2022-01-19 18-16-56](https://user-images.githubusercontent.com/83503327/150329650-70c4b2d1-abb9-4eff-b04e-06b598d6a42b.png)
